docs/source/contributing.rst

```diff
@@ -28,65 +28,65 @@ Contributing
 Contributions to SCICO are welcome. Before starting work, please contact the maintainers, either via email or the GitHub issue system, to discuss the relevance of your contribution and the most appropriate location within the existing package structure.
 
 
-
 .. _installing_dev:
 
 Installing a Development Version
```

Comment:
I think these steps are currently in two places: here and in the "Installing SCICO" section. I suggest we remove the duplication. Leave the steps here, remove them from the installation section, add a pointer from there to here.

Reply:
That's a nice suggestion. It was odd to me at first that they were sorta duplicated in a way, but I didn't want to mess anything up. I can fix that up if given the go-ahead.

Reply:
Agreed with @Michael-T-McCann : remove from the installing scico section.